src/python/WMComponent/DBS3Buffer/DBSBufferDataset.py

```diff
@@ -17,9 +17,8 @@ class DBSBufferDataset(WMConnectionBase, dict):
     information
     """
 
-    def __init__(self, path, id = -1,
-                 processingVer = None, acquisitionEra = None, validStatus = None,
-                 globalTag = None, parent = None, prep_id = None):
+    def __init__(self, path, id=-1,processingVer=None, acquisitionEra=None, validStatus=None,
+                 globalTag=None, parent=None, prep_id=None, stepChain=0):
```

Comment:
Minor, but will be helpful in the future :) How about renaming it to `is_stepchain`?